web/web/settings.py

```diff
@@ -181,7 +181,7 @@
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "DIRS": [
-            "templates",
+            os.path.join(CUCKOO_PATH, "web", "templates"),
```

Comment:
Tests were failing to locate the `templates` directory until this change was made.